packages/bedrock-sdk/src/index.ts

```diff
@@ -1,4 +1 @@
-import { AnthropicBedrock } from './client';
-
-export default AnthropicBedrock;
-export { AnthropicBedrock };
+export { AnthropicBedrock, AnthropicBedrock as default } from './client';
```

Comment:
aside: we should probably export a couple more things here, e.g. `ClientOptions`?